liiweb/views/legislation.py

```diff
@@ -30,7 +32,9 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
 
         qs = self.filter_queryset(self.get_queryset())
-        qs = qs.prefetch_related("taxonomies", "taxonomies__topic")
+        qs = qs.prefetch_related(
+            "taxonomies", "taxonomies__topic", "work", "work__languages"
```

Comment:
`work__languages` isn't a relationship, it's just an attribute on `work`. It will be fetched when the work is fetched.